projects/plugins/jetpack/tests/e2e/lib/pages/homepage.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Internal dependencies
+ */
+import WpPage from './wp-page';
+import logger from '../logger';
+import { searchResultForTest1, searchResultForTest2 } from '../search-helper';
+
+export default class Homepage extends WpPage {
+	static SEARCH_API_PATTERN = /^https:\/\/public-api\.wordpress.com\/rest\/v1.3\/sites\/\d+\/search.*/;
+	static WAIT_FOR_ANIMATION_AND_RENDERING_TIMEOUT = 2000;
```

Comment:
We probably can get rid of this static since it is used only once.

Reply:
removed in https://github.com/Automattic/jetpack/pull/19447/commits/aa9d169dc519eb80e1f98b30bc288adf4e4e185b